lib/omise/OmiseCharge.php

```diff
@@ -3,88 +3,107 @@
+    {
+        return parent::g_retrieve(get_class(), self::getUrl($id), $publickey, $secretkey);
+    }
+
+    /**
+     * (non-PHPdoc)
+     *
+     * @see OmiseApiResource::g_reload()
+     */
+    public function reload()
+    {
+        if ($this['object'] === 'charge') {
+            parent::g_reload(self::getUrl($this['id']));
+        } else {
+            parent::g_reload(self::getUrl());
+        }
+    }
 
-  /**
-   * (non-PHPdoc)
-   * @see OmiseApiResource::g_reload()
-   */
-  public function reload() {
-    if($this['object'] === 'charge') {
-      parent::g_reload(self::getUrl($this['id']));
-    } else {
-      parent::g_reload(self::getUrl());
+    /**
+     * Creates a new charge.
+     *
+     * @param  array  $params
+     * @param  string $publickey
+     * @param  string $secretkey
+     *
+     * @return OmiseCharge
+     */
+    public static function create($params, $publickey = null, $secretkey = null)
+    {
+        return parent::g_create(get_class(), self::getUrl(), $params, $publickey, $secretkey);
     }
-  }
 
-  /**
-   * Creates a new charge.
-   * @param array $params
-   * @param string $publickey
-   * @param string $secretkey
-   * @return OmiseCharge
-   */
-  public static function create($params, $publickey = null, $secretkey = null) {
-    return parent::g_create(get_class(), self::getUrl(), $params, $publickey, $secretkey);
-  }
+    /**
+     * (non-PHPdoc)
+     *
+     * @see OmiseApiResource::g_update()
+     */
+    public function update($params)
+    {
+        parent::g_update(self::getUrl($this['id']), $params);
+    }
 
-  /**
-   * (non-PHPdoc)
-   * @see OmiseApiResource::g_update()
-   */
-  public function update($params) {
-    parent::g_update(self::getUrl($this['id']), $params);
-  }
+    /**
+     * Captures a charge.
+     *
+     * @return OmiseCharge
+     */
+    public function capture()
+    {
+        $result = parent::execute(self::getUrl($this['id']).'/capture', parent::REQUEST_POST, parent::getResourceKey());
+        $this->refresh($result);
 
-  /**
-   * Captures a charge.
-   * @return OmiseCharge
-   */
-  public function capture() {
-    $result = parent::execute(self::getUrl($this['id']).'/capture', parent::REQUEST_POST, parent::getResourceKey());
-    $this->refresh($result);
+        return $this;
+    }
 
-    return $this;
-  }
+    /**
+     * Reverses a charge.
+     *
+     * @return OmiseCharge
+     */
+    public function reverse()
+    {
+        $result = parent::execute(self::getUrl($this['id']).'/reverse', parent::REQUEST_POST, parent::getResourceKey());
+        $this->refresh($result);
 
-  /**
-   * Reverses a charge.
-   * @return OmiseCharge
-   */
-  public function reverse() {
-    $result = parent::execute(self::getUrl($this['id']).'/reverse', parent::REQUEST_POST, parent::getResourceKey());
-    $this->refresh($result);
+        return $this;
+    }
 
-    return $this;
-  }
+    /**
+     * list refunds
```

Comment:
@nam I realize it may ridiculous but sometimes you have `.` sometimes it's not 😂